src/main/java/uk/gov/hmcts/ccd/domain/service/createcase/AuthorisedCreateCaseOperation.java

```diff
@@ -32,19 +35,24 @@
     private final CaseDefinitionRepository caseDefinitionRepository;
     private final AccessControlService accessControlService;
     private final CaseAccessService caseAccessService;
-
+    private SupplementaryDataUpdateOperation supplementaryDataUpdateOperation;
+    private SupplementaryDataUpdateRequestValidator validator;
 
     public AuthorisedCreateCaseOperation(@Qualifier("classified") final CreateCaseOperation createCaseOperation,
                                          @Qualifier(CachedCaseDefinitionRepository.QUALIFIER)
                                          final CaseDefinitionRepository caseDefinitionRepository,
                                          final AccessControlService accessControlService,
-                                         final CaseAccessService caseAccessService) {
+                                         final CaseAccessService caseAccessService,
+                                         @Qualifier("authorised")
```

Comment:
I think we could probably just use the `default` one? Then move this code into the `DefaultCreateCaseOperation`. As it's part of the create case, the user must already have access to the case so no need to go to `AuthorisedSupplementaryDataUpdateOperation` and get the case details again to verify if they have access?

Reply:
done the change